awx/main/migrations/0159_alter_schedule_rrule.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.12 on 2022-04-06 15:08
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0158_make_instance_cpu_decimal'),
```

Comment:
I just merged a migration which will cause a conflict, so you have to bump

Reply:
Fixed.